test/fixtures/rules/std_rule_cases/L030.yml

```
@@ -16,6 +16,16 @@ test_2:
       L030:
         capitalisation_policy: lower
 
+test_bare_functions:
+  fail_str: SELECT current_timestamp from table
+
+  fix_str: SELECT CURRENT_TIMESTAMP from table
```

Might be good to mix some test cases, to ensure your early return does still cover both (I tested and it does btw):

```suggestion
test_bare_functions:
  fail_str: SELECT current_timestamp, min(a) from table

  fix_str: SELECT CURRENT_TIMESTAMP, MIN(a) from table
```

---

Yeah, sure. I think the _get_fix is per segment anyway, but no harm in the extra tests. I've added it as a second test case